baselines/audioworklet.generated.d.ts

```diff
@@ -39,7 +39,7 @@ interface MessageEventInit<T = any> extends EventInit {
 }
 
 interface PromiseRejectionEventInit extends EventInit {
-    promise: Promise<any>;
+    promise: any;
```

Comment:
This is fishy. I did not found a corresponding change in BCD...

Reply:
This is only IDL-side change just to skip JS-to-IDL conversion steps, from TS side we should ignore this. See https://github.com/whatwg/html/pull/9897.